nsis/gvim.nsi

```diff
@@ -120,6 +121,7 @@ CRCCheck force
 SetCompressor /SOLID lzma
 SetCompressorDictSize 64
 ManifestDPIAware true
+XPStyle on
```

Comment:
what does this do please?

Reply:
Oh, it was experimenting with the look and feel and apparently it's still there. It can be removed, of course, but it doesn't have much effect, and the installer dialogs look a bit better.

Reply:
This has no effects because we use MUI2, and it was removed in af610b8e66a7fefcb8db07ee65686f51e1cb288a.

Reply:
Yes, Takata-san, I tested how it affects the appearance and sure enough, it doesn't. I deleted it already. Thanks for the link to the commit.